foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/helper/ConversionManager.java

```diff
@@ -44,6 +49,42 @@
  */
 public class ConversionManager extends CoreConversionManager implements Serializable, Cloneable {
     protected Map defaultNullValues;
+    private static final ZoneId defaultZoneOffset;
+    
+    static {
+        ZoneId tzoneid = null;
+        try {
+            String tzone = PrivilegedAccessHelper.getSystemProperty(SystemProperties.CONVERSION_USE_TIMEZONE);
+            if (tzone != null) {
+                
+                try {
+                    tzoneid = java.time.ZoneId.of(tzone);
+                } catch (Throwable t) {
+                    // If an invalid time zone id is supplied, then fall back to checking for checking for using
```

Comment:
I feel like there should be some sort of warning here to inform the user that the property value they supplied is invalid.

Reply:
Addressed, please re-review.